doc/languages-frameworks/dotnet.section.md

```diff
@@ -34,7 +34,7 @@ mkShell {
 }
 ```
 
-This will produce a dotnet installation that has the dotnet 3.1, 3.0, and 2.1 sdk. The first sdk listed will have it's cli utility present in the resulting environment. Example info output:
+This will produce a dotnet installation that has the dotnet 3.1, 3.0, and 2.1 sdk. The first sdk listed will have its cli utility present in the resulting environment. Example info output:
```

Comment:
```suggestion
This will produce a dotnet installation that has the dotnet 3.1, 3.0, and 2.1 SDK. The first SDK listed will have its CLI utility present in the resulting environment. Example info output:
```